workbench/_api/routes/generate.py

```
@@ -0,0 +1,115 @@
+    if state.remote:
+        return {"job_id": output}
+
+    prompt_tokens, completion_tokens = output
+    return {"data": process_generation(prompt_tokens, completion_tokens, model.tokenizer)}
+
+
+@router.post("/results/{job_id}", response_model=GenerateResponse)
+async def collect_generate(
+    job_id: str,
+    req: GenerateRequest,
+    state: AppState = Depends(get_state),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)